cli/flags.rs

```diff
@@ -389,13 +407,13 @@ impl Flags {
 impl From<Flags> for PermissionsOptions {
```

Comment:
Instead of having this as an `impl From<Flags>` we could instead introduce a `permissions_options()` method on `Flags` that returns this information from a reference of `Flags`.

Reply:
Yep, I think its a good idea